projects/npm-tools/packages/npm-scripts/contrib/prettier/prettier.js

```diff
@@ -1106,12 +1133,40 @@ function prepare(filepath) {
 			file = fallback;
 		}
 	}
-	else {
-		root = getFrontendProjectsRoot(filepath);
 
-		if (root) {
+	const portalRoot = getPortalRoot(filepath);
+
+	if (portalRoot) {
+		const modules = path.join(portalRoot, 'modules/node_modules');
+
+		let scripts = path.join(modules, '@liferay/npm-scripts');
+
+		let wrapper = path.join(scripts, 'src/scripts/prettier.js');
+
+		if (!fs.existsSync(wrapper)) {
+			scripts = path.join(modules, 'liferay-npm-scripts');
+
+			wrapper = path.join(scripts, 'src/scripts/prettier.js');
+		}
+
+		const fallback = path.join(modules, 'prettier/bin-prettier.js');
+
+		if (fs.existsSync(wrapper)) {
+			dir = scripts;
+			file = wrapper;
+		}
+		else if (fs.existsSync(fallback)) {
+			dir = modules;
+			file = fallback;
+		}
+	}
+
+	const frontendProjectsRoot = getFrontendProjectsRoot(filepath);
+
+	if (frontendProjectsRoot) {
```

Comment:
Is this if duplicated? Or is GH UI playing with me? :thinking:

Reply:
It is duplicated, gonna fix

<img src="https://media3.giphy.com/media/26tP4gFBQewkLnMv6/giphy.gif"/>